.ci_support/migrations/python314.yaml

```diff
@@ -1,26 +1,24 @@
+# this is intentionally sorted before the 3.13t migrator, because that determines
+# the order of application of the migrators; otherwise we'd have to add values for
+# is_freethreading and is_abi3 keys here, since that migration extends the zip;
```

Comment:
Wait, I thought conda forge didn't support abi3 builds. This would be a game changer, and I want to inquire a bit before I merge